lib/ood_core/job/adapters/linux_systemd/templates/script_wrapper.erb.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+SSH_HOSTS=(<%= ssh_hosts.join(' ').to_s %>)
+hostnames=`hostname -A`
+for host in ${SSH_HOSTS[@]}
+do
+    if [[ " ${hostnames[@]} " =~ " ${host} " ]]; then
+        hostname=$host
+    fi
+done
+
+if [ -z "$hostname" ]; then
+    printf >&2 "ERROR: Can't start job on [${hostnames[@]}] because it does not match any hostname configured \nin ssh_hosts [${SSH_HOSTS[@]}]. The output of 'hostname -A' must match an entry in ssh_hosts \nfrom the cluster configuration."
+    exit 1
+fi
+
+echo ""
+echo "HOSTNAME:$hostname"
+
+# we need this user to be enabled for lingering or else the newly started
+# service will end as soon as the ssh session starting has exited
+loginctl enable-linger
+
+# Put the script into a temp file on localhost
+systemd_service_tmp_file_base=$(mktemp -d "<%= workdir %>/XXXXXXXXXXXXX" --suffix '_systemd')
```

Comment:
This doesn't need to be random, it's already in a staged root directory that has a UID in it like `~ondemand/data/batch_connect/dev/systemd_test/output/16c55f46-5f5e-43d8-8853-774b5a1eab7f`.

Reply:
Hello, sorry for the delay - got distracted and forgot... 

Anyway - yeah, I pretty much knew that but I figured since linux_host does something similar I should do the same here. Maybe it was needed for testing or some other reason I wasn't familiar with. Easy enough to remove/simplify and can do that if told.

Reply:
Originally they were just placed in `$HOME` so I think they needed to be random.  I think I patched it after the fact to drop them in `staged_root` for debugging purposes.

Now that you mention it, I doubt they need to be randomized either. I think making a directory like `<%=workdir %>.unit_files` is fine.

Reply:
Done - thanks.